### DISCUSSION TITLE
(Help) Needing help from Ben to recover one very important DreamBooth Colab 🙏

### DISCUSSION THREAD

**Discussion by meunumerotim333** [TheLastBen/fast-stable-diffusion#1811] [Q&A]
Needing a looot of help to recover this old colab, I don't like the results from the updated colab. I would appreciate a lot if Ben could just make this exactly colab work:

https://colab.research.google.com/drive/1uUUGu5bH1u2RML-XYm01E9ME3KngpZmW?usp=share_link

Can you help me please? It's the base from my instagram images. If I couldn't keep generating the same results I'm used to I'll lost the will to keep it. If you could keep the same code, but just change enought to get working, I would appreciate 🙏

I'm getting this error after the update:

![image](https://user-images.githubusercontent.com/121369033/227058091-77a5d035-a9ff-40c0-b0d4-5c441eec4dc6.png)

Please, please help 🙏

**Comment by TheLastBen** [comment]
try replacing the old dependencies cell with the new one, it should work

**Reply by meunumerotim333** [reply]
> try replacing the old dependencies cell with the new one, it should work

Thank you, its working. I did this earlier, but for some reason I was distracted and become desperate. Because I swear that was already done. Sorry buddy.
